### PR TITLE
Sea salt aerosol debromination as an option -- off by default

### PR DESCRIPTION
**Pull request by msulprizio**
In this PR, sea salt aerosol debromination is turned off and is include as an option set in input.geos. instead. This came out of concern over benchmark where ozone concentrations are too low, especially over the Southern Ocean.

This addresses issue https://github.com/geoschem/geos-chem/issues/1145. See the discussion there for more details.

### CODE REVIEW

File: KPP/fullchem/fullchem_HetStateFuncs.F90 [hunk 1]
```diff
@@ -314,6 +315,14 @@ SUBROUTINE Halide_Conc( I, J, L, H )
     ! Get halide conc's in cloud (gas-phase, fine & coarse sea salt)
     !=======================================================================
 
+    ! If sea salt aerosol debromination is off, set BrSALA and BrSALC
+    ! concentrations to a small number. This will essentially zero out
+    ! the rates for all reactions involving those species.
+    IF ( .not. H%SSA_debrom ) THEN
+       C(ind_BrSALA) = 1.0e-20_dp
+       C(ind_BrSALA) = 1.0e-20_dp
```
[the remark this answers is not in the document]
Comment (msulprizio) [reply KPP/fullchem/fullchem_HetStateFuncs.F90:323]: Ah! Good catch. Fixing this now.

**Review by yantosca — CHANGES_REQUESTED** [review]
This is a straightforward fix that adds a switch to input.geos to toggle SSA debromination on or off within the heterogeneous chemistry.  Except for the typo already commented on, this is good to release.

File: KPP/fullchem/fullchem_HetStateFuncs.F90 (hunk 1 above)
[the remark this answers is not in the document]
Comment (msulprizio) [reply KPP/fullchem/fullchem_HetStateFuncs.F90]: Now fixed in commit https://github.com/geoschem/geos-chem/commit/8bbe1ddccb06039be1651dca7d65214bebc6a14e.

**Review by yantosca — APPROVED** [review]
OK to merge pending fix of typo.

**Comment by msulprizio** [comment]
This branch passes all GCClassic integration tests:

```
Summary of test results:
------------------------------------------------------------------------------
Execution tests passed: 48
Execution tests failed: 0
Execution tests not yet completed: 0
```

**Comment by msulprizio** [comment]
I am closing this PR. It is now superseded by #1177.

**Pull request closed without merging**